packages/expo-camera/android/src/main/java/expo/modules/camera/CameraViewModule.kt

```diff
@@ -46,6 +48,10 @@ class CameraViewModule : Module() {
 
     Events("onModernBarcodeScanned")
 
+    Property("isModernBarcodeScannerAvailable") {
```

Comment:
Should we add a quick comment here for future reference?

```suggestion
    // Aligned with iOS which has the same property. Will always be true on Android since 
    // the Google code scanner's min sdk is 21 - and Expo's min sdk is set to 25.
    Property("isModernBarcodeScannerAvailable") {
```

Reply:
Yep, makes sense, will do 👍